modules/ROOT/pages/dynamic-evaluate-component-reference.adoc

```diff
@@ -4,29 +4,49 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, ee, esb, studio, dynamic, evaluate, expression
 
-*Available since 4.1*
-
-This Core component evaluates an expression that should result in another script, then evaluates that script for a final result. Such behavior allows you to dynamically select the script, instead of hardcoding it through the xref:transform-component-about.adoc[Transform Message Component].
+The Dynamic Evaluate component evaluates an expression to select a DataWeave script, and then executes the new script to generate a final result. This behavior enables you to dynamically select the script, instead of hardcoding it into the xref:transform-component-about.adoc[Transform Message Component].
```

Comment:
```suggestion
The Dynamic Evaluate component evaluates an expression to select a DataWeave script, and then executes the new script to generate a final result. This behavior enables you to dynamically select the script, instead of hardcoding it into the xref:transform-component-about.adoc[Transform Message component].
```

Reply:
Not sure we need "final" before "result."